lib/ts/Makefile.am

```diff
@@ -271,7 +273,8 @@ test_tslib_SOURCES = \
 	unit-tests/test_Scalar.cc \
 	unit-tests/test_string_view.cc \
 	unit-tests/test_TextView.cc \
-	unit-tests/test_MemSpan.cc
+	unit-tests/test_MemSpan.cc \
```

Comment:
Do me a favor and move both `MemSpan` and `MemArena` so the test source files are in alphabetically order.

Reply:
done.